extensions/http/src/jolie/net/HttpProtocol.java

```diff
@@ -202,6 +203,7 @@
 		private static final String CONTENT_DISPOSITION = "contentDisposition";
 		private static final String DROP_URI_PATH = "dropURIPath";
 		private static final String CACHE_CONTROL = "cacheControl";
+		private static final String STRING_RAW_CONTENT = "stringRawContent";
```

Comment:
Thank you both.

Mmh, then I'd go for something like `.forceContentDecoding:string` (since we're decoding, not encoding) that can be set to either "string" or "raw". That way, it is similar to parameter `format`. Unless there are reasons for thinking that this is different, in that case we could use a sub-subnode as @klag said to specify extra parameters?